packages/compass-workspaces/src/stores/workspaces.ts

```diff
@@ -398,23 +459,12 @@ const reducer: Reducer<WorkspacesState> = (
   }
 
   if (isAction<CloseTabAction>(action, WorkspacesActions.CloseTab)) {
-    const tabToClose = state.tabs[action.atIndex];
-    const tabIndex = state.tabs.findIndex((tab) => tab.id === tabToClose?.id);
-    const newTabs = [...state.tabs];
-    newTabs.splice(action.atIndex, 1);
-    const newActiveTabId =
-      tabToClose.id === state.activeTabId
-        ? // We follow standard browser behavior with tabs on how we handle
-          // which tab gets activated if we close the active tab. If the active
-          // tab is the last tab, we activate the one before it, otherwise we
-          // activate the next tab.
-          (state.tabs[tabIndex + 1] ?? newTabs[newTabs.length - 1])?.id ?? null
-        : state.activeTabId;
-    return {
-      ...state,
-      activeTabId: newActiveTabId,
-      tabs: newTabs,
-    };
+    return _bulkTabsClose({
```

Comment:
Drive-by: at some point this logic was factored out to a separate method, but this place that was the original place for this logic wasn't updated to use it, fixed